extensions/wikia/AdEngine/js/video/uapVideo.js

```diff
@@ -65,35 +74,42 @@ define('ext.wikia.adEngine.video.uapVideo', [
 					'closeButton',
 					'toggleAnimation'
 				], {
-					image: imageContainer,
+					image: providerContainer,
 					container: adSlot,
 					aspectRatio: params.aspectRatio,
 					videoAspectRatio: params.videoAspectRatio
 				});
 
 				video.addEventListener('wikiaAdStarted', function () {
-					var slotWidth = getSlotWidth(adSlot);
-					video.resize(slotWidth, getVideoHeight(slotWidth, params.videoAspectRatio));
+					var size = getVideoSize(adSlot, params);
+					video.resize(size.width, size.height);
 				});
+
 				if (params.autoPlay) {
-					var slotWidth = getSlotWidth(adSlot);
-					video.play(slotWidth, getVideoHeight(slotWidth, params.videoAspectRatio));
+					var size = getVideoSize(adSlot, params);
+					video.play(size.width, size.height);
 				}
 
 				return video;
 			});
 	}
 
+	function getPercentageVideoHeight(size) {
+		return 100 / (size.width / size.height);
+	}
+
 	function loadVideoAd(params) {
 		var loadedPlayer,
 			providerContainer = adSlot.getProviderContainer(params.slotName),
-			slotContainer = providerContainer.parentNode,
-			videoWidth = getSlotWidth(slotContainer);
+			videoContainer = providerContainer.parentNode,
+			size;
 
 		log(['loadVideoAd params', params], log.levels.debug, logGroup);
 
-		params.width = videoWidth;
-		params.height = getVideoHeight(videoWidth, params.videoAspectRatio);
+		size = getVideoSize(videoContainer, params);
+		params.width = size.width;
+		params.height = size.height;
+		params.videoSize = getPercentageVideoHeight(size);
```

Comment:
it looks like it's not used anywhere - please remove along with `getPercentageVideoHeight`